index.js

```diff
@@ -494,8 +499,8 @@ async function connect(client, requeue, gamemode) {
 		}
 	});
 	ClientSocket.on("CPacketSpawnEntity", packet => {
-		if (ENTITIES[packet.type] == undefined) return;
-		if (packet.motion == undefined) packet.motion = {x: 0, y: 0, z: 0};
+		if (ENTITIES[packet.type] == undefined) {return;}
+		if (packet.motion == undefined) {packet.motion = {x: 0, y: 0, z: 0};}
```

Comment:
```suggestion
		if (ENTITIES[packet.type] == undefined) return;
		if (packet.motion == undefined) packet.motion = {x: 0, y: 0, z: 0};
```
more barakeet removing